### PR TITLE
feat: #276 Make multiple queries

### PR DESCRIPTION
**Pull request by Ekaterina1994**
Added multiple queries
Updated WayColumns for add ability mentors to edit cells in DayReportsTable

### CODE REVIEW

File: mw-webapp/src/logic/wayPage/reportsTable/WayColumns.tsx [hunk 1]
```diff
@@ -28,6 +28,16 @@ import styles from "src/logic/wayPage/reportsTable/WayColumns.module.scss";
...
 const columnHelper = createColumnHelper<DayReport>();
 
+/**
+ * Get user name
+ */
+const getUserName = (users: Map<string, UserPreview>, uuid: string, name: string) => {
+  const mentor = users.get(uuid);
+  const userName = mentor ? mentor.name : name;
+
+  return userName;
```
Comment (tritonJS826) [inline mw-webapp/src/logic/wayPage/reportsTable/WayColumns.tsx:38]: what does happened here? Why to get user name we should throw it as params?

File: mw-webapp/src/utils/getChunkArray.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/**
+ * Split array on subArrays
+ */
+export const getChunksArray = <T>(array: T[], chunkLength: number): T[][] => {
+  const chunksArray = [];
```
Comment (tritonJS826) [inline mw-webapp/src/utils/getChunkArray.ts:5]: it looks here like chunks array has type any[], but it is T[][]

File: mw-webapp/src/utils/getChunkArray.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+  for (let i = 0; i < array.length; i += chunkLength) {
+    chunksArray.push(array.slice(i, i + chunkLength));
```
Comment (tritonJS826) [inline mw-webapp/src/utils/getChunkArray.ts:8]: lets create variable `chunkArray`

**Review by tritonJS826 — APPROVED** [review]

File: mw-webapp/src/utils/getChunkArray.ts (hunk 2 above)
Comment (Ekaterina1994) [reply mw-webapp/src/utils/getChunkArray.ts]: Added type

File: mw-webapp/src/utils/getChunkArray.ts (hunk 3 above)
Comment (Ekaterina1994) [reply mw-webapp/src/utils/getChunkArray.ts]: Created

**Review by dayanych — APPROVED** [review]

**Pull request merged**